### PR TITLE
first round of LGTM fixes

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**
Let's play around a bit with LGTM over the weekend...

https://lgtm.com/projects/g/cockpit-project/cockpit/

### CODE REVIEW

File: tools/test-static-code [hunk 1]
```diff
@@ -122,7 +122,7 @@ else
     # all normal files in test/ and tools/ which are either:
     #   - filename is *.py
     #   - executable matching ^#!/usr/bin/python3
-    PYFILES="$(find test/ tools/ -type f '(' -name '*.py' -o -executable -execdir grep -q '^#!/usr/bin/python3' '{}' ';' ')'  -print)"
+    PYFILES="$(find . -type f '(' -name '*.py' -o -executable -execdir grep -q '^#!/usr/bin/python3' '{}' ';' ')'  -print)"
```
Comment (martinpitt) [inline tools/test-static-code:125]: This currently lives in the "various: pycodestyle changes" commit, but is meant to be in the "test-static-code: check all python files" commit. Please either move it, or squash the two commits together.

File: pkg/base1/cockpit.js [hunk 2]
```diff
@@ -3630,8 +3630,8 @@ function factory() {
          * close function to only work when a problem is
          * indicated.
          */
-        var old_close;
         if (shared) {
+            var old_close = client.close;
```
Comment (martinpitt) [inline pkg/base1/cockpit.js:3634]: Please use `const` here -- scoping of `var` is really broken in JS, it does not do what you think it does; it always applies to the scope of the containing function, not just the `{}` block

**Review by martinpitt — CHANGES_REQUESTED** [review]
Mostly LGTM :grin: Just two trivial things. (and of course your "LGMT" typos in the commit messages) Thanks!

File: pkg/base1/cockpit.js (hunk 2 above)
Comment (allisonkarlitskaya) [reply pkg/base1/cockpit.js]: done, thanks.

I explicitly avoided changing this for exactly this reason.  I knew it was different, and I didn't know which one I wanted :rofl:

File: tools/test-static-code (hunk 1 above)
Comment (allisonkarlitskaya) [reply tools/test-static-code:125]: done, thanks

**Review by martinpitt — APPROVED** [review]
Thank you! As you already enabled LGTM and this is now red on all other PRs, landing this quickly.

**Pull request merged**